frontend/src/lib/contributors.ts

```diff
@@ -23,88 +21,151 @@ export const current: Contributors = {
 			role: 'Product Manager and Backend Engineer',
 			site: 'https://kevwang.dev/',
 			img: {
-				base: path('2022-23/kevin_1.jpg'),
-				silly: path('2022-23/kevin_2.jpg')
+				base: path('kevin_1.jpg'),
+				silly: path('kevin_2.jpg')
 			}
 		},
 		{
 			name: 'Zachary Zollman',
 			role: 'Backend Lead',
 			site: 'https://zacharyzollman.com/',
 			img: {
-				base: path('2022-23/zachary_1.jpg'),
-				silly: path('2022-23/zachary_2.jpg')
+				base: path('zachary_1.jpg'),
+				silly: path('zachary_2.jpg')
 			}
 		},
 		{
 			name: 'Matthew Rowland',
 			role: 'Frontend Lead',
-			site: null,
+			site: 'https://www.linkedin.com/in/matthew-rowland-dev/',
 			img: {
-				base: path('2022-23/matthew_1.jpg'),
-				silly: path('2022-23/matthew_2.jpg')
+				base: path('matthew_1.jpg'),
+				silly: path('matthew_2.jpg')
 			}
 		},
 		{
 			name: 'Michelle Tran',
 			role: 'Design Lead',
 			site: 'https://michelletran.design',
 			img: {
-				base: path('2022-23/michelle_1.jpg'),
-				silly: path('2022-23/michelle_2.jpg')
+				base: path('michelle_1.jpg'),
+				silly: path('michelle_2.jpg')
 			}
 		},
 		{
 			name: 'Henric Zhang',
 			role: 'Frontend Engineer',
 			site: null,
 			img: {
-				base: path('2022-23/henric_1.jpg'),
-				silly: path('2022-23/henric_2.jpg')
+				base: path('henric_1.jpg'),
+				silly: path('henric_2.jpg')
 			}
 		},
 		{
-			name: 'Ethan Ikegami',
-			role: 'Backend Engineer',
-			site: 'https://ethanikegami.com/',
+			name: 'Jaden Moore',
+			role: 'Frontend Engineer',
+			site: 'https://www.jdyn.dev/',
```

Comment:
Your version of this file deviates from the changes made in #566. #566 should be merged first and then you can remove the changes made to `contributors.ts` in this PR, assuming they don't pertain to the real changes you are attempting to make.

Reply:
This version of the file should already include changes from #566 since I merged in that branch into this one and added some modifications (including missing past contributors etc.). I intend for the current `contributors.ts` in this branch with the changes it contains to be merged into master.

Reply:
The merge wasn't encompassing because for example, I removed my site url, but in your merge, it was added back. I also reordered the json. imo the design team members should be moved up in the ordering.

Reply:
Ah I see. Could you give me a list of changes you'd like and I can implement them? Alternatively feel free to commit to this branch with any changes you see fit. I would just keep the grouping of the Frontend/Backend/Design members together for consistency.